# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/image_util/copyimage.inl
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/image_util/copyimage.inl
--- a/gfx/angle/checkout/src/image_util/copyimage.inl	Sat Sep 07 13:36:46 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,38 +0,0 @@
-//
-// Copyright (c) 2014 The ANGLE Project Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-//
-
-// copyimage.inl: Defines image copying functions
-
-namespace angle
-{
-template <typename sourceType, typename colorDataType>
-void ReadColor(const uint8_t *source, uint8_t *dest)
-{
-    sourceType::readColor(reinterpret_cast<Color<colorDataType>*>(dest),
-                          reinterpret_cast<const sourceType*>(source));
-}
-
-template <typename destType, typename colorDataType>
-void WriteColor(const uint8_t *source, uint8_t *dest)
-{
-    destType::writeColor(reinterpret_cast<destType*>(dest),
-                         reinterpret_cast<const Color<colorDataType>*>(source));
-}
-
-template <typename SourceType>
-void ReadDepthStencil(const uint8_t *source, uint8_t *dest)
-{
-    SourceType::ReadDepthStencil(reinterpret_cast<DepthStencil *>(dest),
-                                 reinterpret_cast<const SourceType *>(source));
-}
-
-template <typename DestType>
-void WriteDepthStencil(const uint8_t *source, uint8_t *dest)
-{
-    DestType::WriteDepthStencil(reinterpret_cast<DestType *>(dest),
-                                reinterpret_cast<const DepthStencil *>(source));
-}
-}  // namespace angle